### PR TITLE
test circleci integration

### PR DESCRIPTION
**Pull request by fdsimms**

### CODE REVIEW

**Comment by fdsimms** [comment]
<details>
<summary>
<b>1 failed tests 😱</b>
</summary>
---

**basic test**
  <details>
  <summary>
    See what went wrong
  </summary>

```bash
Error: expect(received).toBe(expected) // Object.is equality

Expected: 16
Received: 11
    at Object.<anonymous>.test (/home/circleci/ci-test-failure-reporter/src/index.test.ts:2:21)
    at Object.asyncJestTest (/home/circleci/ci-test-failure-reporter/node_modules/jest-jasmine2/build/jasmineAsyncInstall.js:102:37)
    at resolve (/home/circleci/ci-test-failure-reporter/node_modules/jest-jasmine2/build/queueRunner.js:41:12)
    at new Promise (<anonymous>)
    at mapper (/home/circleci/ci-test-failure-reporter/node_modules/jest-jasmine2/build/queueRunner.js:26:19)
    at promise.then (/home/circleci/ci-test-failure-reporter/node_modules/jest-jasmine2/build/queueRunner.js:71:41)
    at process._tickCallback (internal/process/next_tick.js:68:7)
```
  </details>
---
  
</details>

**Comment by postlight-org** [comment]
<details>
<summary>
<b>1 failed tests 😱</b>
</summary>
---

**basic test**
  <details>
  <summary>
    See what went wrong
  </summary>

```bash
Error: expect(received).toBe(expected) // Object.is equality

Expected: 16
Received: 11
    at Object.<anonymous>.test (/home/circleci/ci-test-failure-reporter/src/index.test.ts:2:21)
    at Object.asyncJestTest (/home/circleci/ci-test-failure-reporter/node_modules/jest-jasmine2/build/jasmineAsyncInstall.js:102:37)
    at resolve (/home/circleci/ci-test-failure-reporter/node_modules/jest-jasmine2/build/queueRunner.js:41:12)
    at new Promise (<anonymous>)
    at mapper (/home/circleci/ci-test-failure-reporter/node_modules/jest-jasmine2/build/queueRunner.js:26:19)
    at promise.then (/home/circleci/ci-test-failure-reporter/node_modules/jest-jasmine2/build/queueRunner.js:71:41)
    at process._tickCallback (internal/process/next_tick.js:68:7)
```
  </details>
---
  
</details>

**Comment by postlight-org** [comment]
<details>
<summary>
<b>1 failed tests 😱</b>
</summary>
---

**basic test**
  <details>
  <summary>
    See what went wrong
  </summary>

```bash
Error: expect(received).toBe(expected) // Object.is equality

Expected: 16
Received: 11
    at Object.<anonymous>.test (/home/circleci/ci-test-failure-reporter/src/index.test.ts:2:21)
    at Object.asyncJestTest (/home/circleci/ci-test-failure-reporter/node_modules/jest-jasmine2/build/jasmineAsyncInstall.js:102:37)
    at resolve (/home/circleci/ci-test-failure-reporter/node_modules/jest-jasmine2/build/queueRunner.js:41:12)
    at new Promise (<anonymous>)
    at mapper (/home/circleci/ci-test-failure-reporter/node_modules/jest-jasmine2/build/queueRunner.js:26:19)
    at promise.then (/home/circleci/ci-test-failure-reporter/node_modules/jest-jasmine2/build/queueRunner.js:71:41)
    at process._tickCallback (internal/process/next_tick.js:68:7)
```
  </details>
---
  
</details>

File: src/index.test.ts [hunk 1]
```diff
@@ -1,3 +1,3 @@
 test('basic test', () => {
-  expect(2 + 3 + 6).toBe(12);
+  expect(2 + 3 + 6).toBe(16);
```
Comment (adampash) [inline src/index.test.ts:2]: How does it handle if the tests are inside of a describe block? I think what we'd want is to have each describe section expand into its sub-tests, also.

**Comment by postlight-org** [comment]
<details>
<summary>
<b>1 failed tests 😱</b>
</summary>
---

**basic test**
  <details>
  <summary>
    See what went wrong
  </summary>

```bash
Error: expect(received).toBe(expected) // Object.is equality

Expected: 16
Received: 11
    at Object.toBe (/home/circleci/ci-test-failure-reporter/src/index.test.js:2:21)
    at Object.asyncJestTest (/home/circleci/ci-test-failure-reporter/node_modules/jest-jasmine2/build/jasmineAsyncInstall.js:102:37)
    at resolve (/home/circleci/ci-test-failure-reporter/node_modules/jest-jasmine2/build/queueRunner.js:41:12)
    at new Promise (<anonymous>)
    at mapper (/home/circleci/ci-test-failure-reporter/node_modules/jest-jasmine2/build/queueRunner.js:26:19)
    at promise.then (/home/circleci/ci-test-failure-reporter/node_modules/jest-jasmine2/build/queueRunner.js:71:41)
    at process._tickCallback (internal/process/next_tick.js:68:7)
```
  </details>
---
  
</details>

**Pull request merged**